client/src/components/ActiveChat/Messages.js

```diff
@@ -4,15 +4,21 @@ import { SenderBubble, OtherUserBubble } from "../ActiveChat";
 import moment from "moment";
 
 const Messages = (props) => {
-  const { messages, otherUser, userId } = props;
+  const { messages, lastReads, otherUser, userId } = props;
+
+  const otherUsersWithLastReads = (messageId, lastReads) => {
+    return lastReads.map(read => {
+      return read.messageId === messageId && read.userId !== userId && otherUser;
+    });
+  }
```

Comment:
This loop will also run every time the component re-renders. There's a more efficient way of having a boolean to conditionally render the other user's avatar.